package.json

```diff
@@ -16,8 +16,8 @@
     "dev": "lerna run --parallel dev",
     "test:components": "lerna exec --scope @carto/airship-components -- npm run test",
     "test:components:watch": "lerna exec --scope @carto/airship-components -- npm run test:watch",
-    "lint": "lerna run lint",
-    "lint:fix": "lerna run lint:fix",
+    "lint": "lerna exec npm run lint",
```

Comment:
lerna forwards error code by default. I suspect that the failure comes from https://github.com/CartoDB/airship/blob/master/packages/components/package.json#L19. Shouldn't it be `&&`?

Reply:
🤦‍♂️ totally right

Reply:
With **&** it should launch both processes in parallel. If **&&** gets along with lerna, then let's use it and go for the serial execution

Reply:
Yes, but I think `&` doesn't forward the error code properly to npm 🤷‍♂️

Reply:
yes, we ned to use &&

Reply:
so be it